ui/ducks/send/send.js

```diff
@@ -843,6 +845,10 @@ const slice = createSlice({
         ):
           state.status = SEND_STATUSES.INVALID;
           break;
+        case state.asset.type === ASSET_TYPES.TOKEN &&
```

Comment:
not req but wondering if it makes sense to separate this out so that asset has an error field, as we do with amount and gas?

Reply:
i was thinking that when I was working through erc721 updates, i think its a definite candidate for an improvement but not to be handled here.